gr-fec/python/fec/qa_ecc_ccsds_27.py

```diff
@@ -36,6 +36,20 @@ def xtest_ccsds_27(self):
         dst_data = dst.data()
         self.assertEqual(expected, dst_data)
 
+    def test_ccsds_27_long(self):
+        src_data = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10] * 100000
```

Comment:
This generates 1,000,000 bytes = 8,000,000 bits of input, which is enough to trigger the bug. The test takes about 650ms to execute on my laptop, which seems acceptable.

Reply:
I agree. This is important enough.